src/lib.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 The peniko authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Also licensed under MIT license, at your choice.
+
+mod blend;
+mod brush;
+mod parse;
+mod style;
+
+/// Re-export of the kurbo 2D curve library.
+pub use kurbo;
+
+pub use blend::*;
+pub use brush::*;
+pub use parse::ParseError;
```

Comment:
If the only thing in there is an error, maybe the module should be named "error"? I'd expect parsing code. Minor nit though.